### ISSUE TITLE
TypeError -  Unhandled Promise Rejection: Failed to fetch

### ISSUE THREAD

**Issue by gndk** [newrelic/newrelic-browser-agent#622]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

Page timing POST returns error, but NR script doesn't handle it properly.

## Steps to Reproduce

New Relic server returns 500 for page timing POST.

## Expected Behavior

Proper error handling in New Relic script.

## Relevant Logs / Console output

```
TypeError: Unhandled Promise Rejection: Failed to fetch
  at e.<computed>(/profile/view/01857bcb-05fd-71e3-addf-3ddb486b4ce0:5:16380)
  at t(./node_modules/@sentry/utils/esm/instrument.js:142:28)
  at o(/async-api.30bd804e-1.236.0.min.js:2:1266)
  at C._send(/148.1a20d5fe-1.236.0.min.js:2:3697)
  at C.sendX(/148.1a20d5fe-1.236.0.min.js:2:1787)
  at s(/page_view_timing-aggregate.bd6de33a-1.236.0.min.js:1:1442)
  at ? (/page_view_timing-aggregate.bd6de33a-1.236.0.min.js:1:1555)
  at Array.forEach(<anonymous>)
  at c.runHarvest(/page_view_timing-aggregate.bd6de33a-1.236.0.min.js:1:1542)
  at c.unload(/page_view_timing-aggregate.bd6de33a-1.236.0.min.js:1:596)
  at HTMLDocument.<anonymous>(/profile/view/01857bcb-05fd-71e3-addf-3ddb486b4ce0:5:13714)
  at sentryWrapped(./node_modules/@sentry/browser/esm/helpers.js:90:17)
  at HTMLDocument.nrWrapper(/profile/view/01857bcb-05fd-71e3-addf-3ddb486b4ce0:5:14381)
```

<img width="1312" alt="image" src="https://github.com/newrelic/newrelic-browser-agent/assets/25748293/0f418d22-6643-4dfd-bd8f-f9270ae2fc77">

<img width="1336" alt="image" src="https://github.com/newrelic/newrelic-browser-agent/assets/25748293/1a9ad843-ac74-487f-8217-8e7754000a09">

## Your Environment

Latest Chrome on Mac

New Relic PHP Agent v10.11.0.3 with browser auto instrumentation enabled.

No idea what the browser agent version is, but I guess latest as of today. The inserted code mentions `nr-loader-spa-1.236.0.min.js.LICENSE.txt`.

**Comment by patrickhousley** [comment]
@gndk We recently released version 1.237.1 of the browser agent. Could you please let us now if this issue is persisting with that version? You may need to restart your APM applications for it to pickup the new browser agent version.

**Comment by patrickhousley** [comment]
Closing as resolved for now. Please let us know if this issue persists.
